docs/CHANGELOG.md

```diff
@@ -118,6 +118,8 @@ Tasks that support the `evmChainID` parameter are as follows:
 - `estimategaslimit`
 - `ethtx`
 
+For OCR jobs, `ocrDatabaseTimeout` can be specified to override chain-specific default timeout. However, remember that global variable `OCR_DATABASE_TIMEOUT` (if set) overrides chain specific config which, in turn, will override this this job specific parameter.
```

Comment:
I don't think this is quite right - the order of precedence should be (and correctly is based on the current implementation) job spec > global env var > chain specific default

Reply:
Fixed